docs/content/stable/api/ysql/ddl-behavior.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Behavior of DDL statements [YSQL]
+headerTitle: Behavior of DDL statements
+linkTitle: Behavior of DDL statements
+description: Explains how the behavior of DDL statements works in YugabyteDB YSQL and documents differences from Postgres behavior. [YSQL].
```

Comment:
/s/documents differences from Postgres behavior/outlines the between YugabyteDB and PostgreSQL.